stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -808,10 +676,12 @@ end
 #! Check that the Proof-of-Work contained in the nonce is equal to the required number
 #! of bits prescribed by grinding bits. The grinding factor is assumed to be less than 32.
 #!
-#! Input: [grinding_factor, ...]
+#! Input: [...]
 #! Output: [...]
 #! Cycles: 73
 export.check_pow
+    # Load the grinding factor
+    exec.constants::grinding_factor_ptr mem_load
```

Comment:
There are two patterns we seem to be using:

- Sometimes we get the pointer and do `mem_load` (like here).
- Other times the procedure loads the value - e.g., `get_trace_domain_generator`?

Should we try to make things consistent? For example, this could probably have been:
```
exec.constants::get_grinding_factor
```

This is applicable to other places as well.

Reply:
Yes, I have started adding setters and getters for constants when we only need to get or set a field element.
I have now completed this and added getters and setters for all constants appearing in the code.

As for getting and setting words, I haven't done so as we want to have the option of either using `padw` before loading or not if we can overwrite the top word of the stack.